csharp/ql/src/Diagnostics/DiagnosticNoExtractionErrors.ql

```diff
@@ -11,8 +11,5 @@ import csharp
 import semmle.code.csharp.commons.Diagnostics
 
 from File file
-where
-  file.fromSource() and
-  not exists(ExtractorError e | e.getLocation().getFile() = file) and
-  not exists(CompilerError e | e.getLocation().getFile() = file)
+where file.fromSource()
```

Comment:
Perhaps change `@name Successfully extracted files` to `@name Extracted files`, like on https://github.com/github/codeql/pull/15256.